www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+either party breaches the terms of the agreement (e.g. if the Team fails to
+provide the promised services).
+
+  1. Gratipay does not provide refunds. Teams are responsible for producing the
+     Open Work they offer. Gratipay does not hold funds on behalf of Teams, and
+does not warrant or guarantee that Teams will provide the Open Work they
+promise.
+
+1. Payroll Rules
+
+  1. Teams can use the Service's Payroll feature to automatically divide
+     incoming payments among Team Members, according to the allocation set by
+the Members.
+
+  1. Gratipay does not determine how a Team's Payroll is allocated and is not
+     responsible for how Teams internally distribute payments. The agreement
+between Team Members and Owners regarding the allocation of payments is solely
+between those parties.
+
+1. Rules for Gratipay Teams
+
+  1. Teams may accept payments on the Service only for Open Work, meaning that
+     they provide a clear path for any individual to voluntarily begin
+contributing to the Team's work, and to be paid for their contribution.
+
+  1. A Team can only begin receiving payments from Participants once it has
+     provided a working withdrawal mechanism for receiving payments (e.g. bank
+account information).
+
+  1. Teams and their Open Work must be consistent with Gratipay's [Brand
+     Guidelines](http://inside.gratipay.com/big-picture/brand/), Community
+Guidelines, and Acceptable Use Policy.
+
+  1. Gratipay reserves the right to reject, suspend, or remove a Team at any
+     time, and for any reason.
+
+  1. When a new Team is established, the Gratipay community will have the
+     opportunity to evaluate and provide commentary about the Team on
+Gratipay's Github repository. However, all decisions regarding whether to
+accept, reject, suspend, remove, or take any other action regarding a Team
+belong solely to Gratipay.
+
+1. Fees and Taxes
+
+  1. Gratipay does not take any fee from payments made by Participants to
+     Teams. Gratipay and the Service are funded through the Service itself, by
+voluntary contributions from Gratipay Participants and Teams who wish to fund
+our own Open Work. To contribute to Gratipay, please [visit Gratipay's Team
+page](/Gratipay/).
+
+  1. You are responsible for paying any third-party fees associated with your
+     use of the Service, such as credit card and bank fees.
+
+  1. You are responsible for paying any taxes and withholdings associated with
+     your use of the Service. Team Owners, and not Gratipay, are responsible
+for any taxes and withholdings required by applicable law with respect to
+payments made to Team Members (whether through the Payroll feature or
+otherwise), including without limitation payroll tax, unemployment insurance,
+and worker compensation insurance payments. Team Owners are also responsible
+for providing any required tax and other documentation to employees,
+independent contractors, tax authorities, or any other person.
```

Comment:
How about minimum wage law? Probably doesn't fit under Fees and Taxes, but seems like something we want to remind Team Owners that they're responsible for complying with.